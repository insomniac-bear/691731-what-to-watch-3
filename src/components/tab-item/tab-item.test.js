import React from 'react';
import renderer from 'react-test-renderer';
import TabItem from './tab-item.jsx';

const mockData = {
  tabName: `Name`,
  activeTab: `activeTab`,
};

it(`Render Tab item`, () => {
  const tree = renderer
    .create(<TabItem
      tabName={mockData.tabName}
      activeTab={mockData.activeTab}
      tabChangeHandler={()=>{}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
