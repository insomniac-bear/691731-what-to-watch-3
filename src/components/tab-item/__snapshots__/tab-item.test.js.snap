// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tab item 1`] = `
<li
  className="movie-nav__item"
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Name
  </a>
</li>
`;
